x/track.rb

```diff
@@ -34,5 +34,11 @@ def initialize(data)
     def fetch_cmd(problem=problems.first)
       "exercism fetch #{id} #{problem}"
     end
+
+    def find(problem_slug)
```

Comment:
Let's call this just `slug` instead of `problem_slug`, since slugs are always problem/exercise slugs.